.eslintrc.cjs

```diff
@@ -2,7 +2,8 @@
 module.exports = {
   env: {
     browser: true,
-    es2021: true
+    es2021: true,
+    jest: true
```

Comment:
we don't really use jest tho - its vitest

Reply:
<img width="1056" alt="Screenshot 2024-03-12 at 13 50 10" src="https://github.com/reaviz/reablocks/assets/5399389/62470b03-7307-4095-843a-a27f4412fa26">

added it to fix this linter error

Reply:
you can import `describe`, `test`, and `expect` from `vitest` instead. I can push a fix